proxy/.env

```diff
@@ -6,3 +6,4 @@
 
 CLIENT_ID=
 CLIENT_SECRET=
+VISUALIZATION_URL=

```

Comment:
Can you default this to `https://api.bentley.com/imodeljs`?

Reply:
```suggestion
VISUALIZATION_URL=https://api.bentley.com/imodeljs
```